### PR TITLE
chore(insights): rename the value field in multiple breakdowns

### PR DESCRIPTION
**Pull request by skoob13**
## Problem

The PR renames the `value` to `property` field to improve backward compatibility with legacy multiple breakdowns because there are 17k+ saved funnels with the old shape.

## Changes

- Rename the field
- Fix tests
- Refactor cleaning filters to be backward compatible with legacy breakdowns (frontend)
- Refactor conversion of filters to a query (backend)

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Existing unit tests and manual testing.

### CODE REVIEW

File: frontend/__snapshots__/exporter-exporter--funnel-top-to-bottom-breakdown-insight--dark.png
Comment (skoob13) [inline frontend/__snapshots__/exporter-exporter--funnel-top-to-bottom-breakdown-insight--dark.png:1]: Snapshots are correct. They reverted to prior multiple breakdowns in Funnels: https://github.com/PostHog/posthog/commit/7a3e849521e55a860cfc1adbb10e244817221b0c#diff-97ccf0cbc4b249f761838d67a3d9ccc7af51e8d5fbd01f6a871afcb22208c1a7.

**Comment by skoob13** [comment]
@thmsobrmlr a few notes about the PR:
- the `value` field was renamed to `property`
- `filter_to_query` (backend):
  - For Funnels: I reverted my changes here, so it extracts a first breakdown.
  - For Trends: keeps multiple breakdowns.
- `clean_filters` (frontend):
  - For Funnels: keeps breakdowns as they were before.
  - For Trends: keeps multiple breakdowns and only restores the `normalize_url`.

File: frontend/src/scenes/insights/utils/cleanFilters.ts [hunk 1]
```diff
@@ -157,29 +157,20 @@ const cleanBreakdownParams = (cleanedParams: Partial<FilterType>, filters: Parti
...
-                            b.normalize_url && b.value
-                                ? cleanBreakdownNormalizeURL(b.value, filters.breakdown_normalize_url)
-                                : b.normalize_url,
-                    }))
-                    .filter((b) => !!b.value)
-            }
+            cleanedParams['breakdown_type'] = undefined
+            cleanedParams['breakdowns'] = filters.breakdowns.map((b) => ({
+                property: b.property,
+                type: b.type || filters.breakdown_type,
```
Comment (thmsobrmlr) [inline frontend/src/scenes/insights/utils/cleanFilters.ts:163]:
```suggestion
                type: b.type || filters.breakdown_type || 'event',
```

**Review by thmsobrmlr — APPROVED** [review]

**Comment by thmsobrmlr** [comment]
For posterity, this gives us all insights from our team already using the trends multi-breakdowns:
```sql
SELECT filters->>'breakdowns', team_id, * FROM posthog_dashboarditem
WHERE filters->>'breakdowns' IS NOT NULL
AND filters->'breakdowns'->0->>'property' IS NULL
```

File: frontend/__snapshots__/scenes-app-dashboards--insight-legend--dark.png
Comment (Twixes) [inline frontend/__snapshots__/scenes-app-dashboards--insight-legend--dark.png:1]: LGTM if looks good to Thomas, just is it expected for this dashboard to be blank?

Comment (skoob13) [reply frontend/__snapshots__/scenes-app-dashboards--insight-legend--dark.png:1]: Indeed, it has fixed the snapshot.

**Pull request merged**